static/js/services/search.js

```diff
@@ -34,7 +34,8 @@ var _ = require('underscore');
         return false;
       };
 
-      var getPage = function(type, rows, options) {
+      // Get the subset of rows, in appropriate order, according to options.
+      var getPageSubset = function(type, rows, options) {
```

Comment:
A subset of rows is the definition of a "page", hence the term pagination to mean showing a certain number of items at a time. I don't know what a "page subset" is?